extensions/openai/cache_embedding_model.py

```diff
@@ -6,7 +6,6 @@
 import os
 
 import sentence_transformers
-from extensions.openai.script import params
 
-st_model = os.environ.get("OPENEDAI_EMBEDDING_MODEL", params.get('embedding_model', 'all-mpnet-base-v2'))
+st_model = os.environ["OPENEDAI_EMBEDDING_MODEL"] if "OPENEDAI_EMBEDDING_MODEL" in os.environ else "all-mpnet-base-v2"
```

Comment:
I think that it would be better to keep the `.get` syntax here

Reply:
OK